.cloud/.azure/docker.json

```diff
@@ -0,0 +1,13 @@
+{
+    "inference_source_directory": "<your-inference-source-directory>",
+    "inference_entry_script": "<your-inference-entry-script>",
+    "conda_file": "<your-conda-environment-file-path>",
+    "extra_docker_file_steps": "<your-extra-docker-steps-file-path>",
+    "runtime": "<'python' or 'spark-py'>",
+    "enable_gpu": false,
+    "cuda_version": "<your-cuda-version>",
+    "custom_base_image": "<your-custom-docker-base-image>",
+    "description": "<your-webservice-description>",
+    "skip_deployment": true,
```

Comment:
is docker.json file used anywhere? In that case will update the tests likewise

Reply:
Docker.json just is an example of some of the parameters that you can use, if you only want to create a docker image. This can be changed or removed, if it is not necessary.